Packs/IBMSecurityVerify/Integrations/IBMSecurityVerify/IBMSecurityVerify.yml

```diff
@@ -0,0 +1,78 @@
+category: Analytics & SIEM
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: IBMSecurityVerify
+  version: -1
+configuration:
+- display: Server URL
+  name: url
+  required: true
+  type: 0
+  section: Connect
+  additionalinfo: 'For example: https://tenant.verify.ibm.com'
+- display: Client ID
+  name: credentials
+  required: true
+  section: Connect
+  type: 9
+  displaypassword: Client Secret
+- display: The maximum number of events per fetch
+  name: max_fetch
+  additionalinfo: The maximum is 50,000.
+  defaultvalue: 10000
+  section: Collect
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+  section: Connect
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+  section: Connect
+description: IBM Security Verify provides a secure and scalable solution for collecting and managing security events from IBM Security Verify, offering advanced threat detection and response capabilities for protecting identities, applications, and data.
+display: IBM Security Verify
+name: IBMSecurityVerify
+script:
+  commands:
+  - name: ibm-security-verify-get-events
+    description: Retrieves events from IBM Security Verify.
+    arguments:
+    - name: should_push_events
+      description: If set to 'True', the command will create events; otherwise, it will only display them.
+      defaultValue: 'False'
+      auto: PREDEFINED
+      isArray: false
+      predefined:
+      - 'True'
+      - 'False'
+    - name: limit
+      description: Maximum number of results to return.
+      defaultValue: 1000
+    - name: last_id
+      description: "The ID of the last event retrieved. Use together with `last_time` for pagination to get events after this ID. Example: 1234abcd-5678-90ef-1234-567890abcdef."
+    - name: last_time
+      description: "The timestamp of the last event retrieved. Use together with `last_id` for pagination to get events after this time. Example: 1672531200000."
+    - name: sort_order
+      description: "Order to sort events by: 'Desc' or 'Asc'."
+      defaultValue: 'Desc'
```

Comment:
why this is the default value?

Reply:
It's a command that doesn't appear in the design, how much to write?